### DISCUSSION TITLE
How to apply monospaced block or quoted text formatting to clipped text

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#86] [Q&A] [answered]
I want apply monospaced block or quoted text formatting to clipped text. How to add that option in the code given below 

![image](https://user-images.githubusercontent.com/67494083/157114010-dcd8239e-1346-430c-8d22-a98bdad7c701.png)

**Comment by buggyj** [comment, marked as answer]
You can apply any tiddlywiki markup, so maybe

\n\n```((*@text*))```\n\n

**Reply by arunnbabu81** [reply]
Can you give an example. I am not able to get it work

**Reply by buggyj** [reply]
monospace could be like this:

![image](https://user-images.githubusercontent.com/2855795/157197978-40b83a6f-8af4-4808-9fe5-17d79f45350b.png)

**Reply by arunnbabu81** [reply]
I used this rule.

![image](https://user-images.githubusercontent.com/67494083/157287823-02bce0b6-0639-4b85-bc5c-6bb300c06425.png)


`@text=((*@removelf(@text)*))
$text=((*$text*)) \n\n<a href="((*@pageRef*))">((*@userstring*))</a>\n\n```\n\n((*@text*))\n\n```\n\n`

And it works well.

![image](https://user-images.githubusercontent.com/67494083/157287769-ec5a96a9-0d46-418e-8081-9d952e356616.png)

**Reply by arunnbabu81** [reply]
Then I used details widget formatting for the clipped text using this rule.


![image](https://user-images.githubusercontent.com/67494083/157288505-9d1dbd8e-adf3-4949-8f3e-cb69251b3e60.png)

`@text=((*@removelf(@text)*))
$text=((*$text*)) \n\n<details><summary><a href="((*@pageRef*))">((*@userstring*))</a>\n\n</summary>\n\n((*@text*))\n\n</details>\n\n`

This also works.
![image](https://user-images.githubusercontent.com/67494083/157288623-cbd335a5-7059-43d5-bcc0-523110dce168.png)

![image](https://user-images.githubusercontent.com/67494083/157288666-901a6c4c-529a-4db5-9845-f3320c7658aa.png)

**Reply by arunnbabu81** [reply]
Then I tried to make a details widget  formatting for the clipped text with code block like appearance (based  on [telmiger's details widget plug in](https://tid.li/tw5/plugins.html#%24%3A%2Fplugins%2Ftelmiger%2Fdetails%2Fabout))

![image](https://user-images.githubusercontent.com/67494083/157289160-adfa2a03-0cfd-46a8-a8c4-54db0a05eef4.png)


`@text=((*@removelf(@text)*))
$text=((*$text*)) \n\n<$details summary="<a href="((*@pageRef*))">((*@userstring*))</a>">\n\n```\n\n((*@text*))\n\n```\n\n</$details>\n\n`


But this is not working

![image](https://user-images.githubusercontent.com/67494083/157289485-d6f299ef-bad7-4e43-91cb-bd67351b5f21.png)

![image](https://user-images.githubusercontent.com/67494083/157289569-1e8ec06b-e942-411b-bbb3-14cbd733401d.png)

@buggyj can you help?

**Reply by buggyj** [reply]
maybe it needs tripple speech marks:

`<$details summary="""<a href="((*@pageRef*))">((*@userstring*))</a>""">`

**Reply by arunnbabu81** [reply]
> `<$details summary="""<a href="((*@pageRef*))">((*@userstring*))</a>""">`

I tried this rule

`$text=<$details summary="""<a href="((*@pageRef*))">((*@userstring*))</a>""">\n\n```\n\n((*@text*))\n\n```\n\n</$details>`

![image](https://user-images.githubusercontent.com/67494083/157303080-eed6f9e3-fd82-4796-ad98-8eb8ca2ede84.png)


But I am not getting it correct.

![image](https://user-images.githubusercontent.com/67494083/157303141-ab071e8a-efbb-4a5e-b453-19cbfb1be07b.png)

![image](https://user-images.githubusercontent.com/67494083/157303192-5372eae0-fc50-4491-ab7f-01eced4c9394.png)

**Reply by buggyj** [reply]
use without the widget:
`
$text=<details> <summary><a href="((*@pageRef*))">((*@userstring*))</a></summary><pre>((*@text*))</pre></details> 
`

**Reply by arunnbabu81** [reply]
> use without the widget:
> `$text=<details> <summary><a href="((*@pageRef*))">((*@userstring*))</a></summary><pre>((*@text*))</pre></details>`

This one works. Thank you @buggyj
